libs/libvtrutil/src/vtr_error.h

```diff
@@ -4,15 +4,32 @@
 #include <stdexcept>
 #include <string>
 
+/**
+ * @file
+ * @brief A utility container that can be used to identify VTR execution errors.
+ * 
+ * The recommended usage is to store error information within this container during an error event and handling the error with exceptions. If the exception is not handled (exception is not caught), this will result in the termination of the program.
```

Comment:
"handling with exceptions" -> "store information in this container about the error, and then throw an exception with the container"

Reply:
Added the requested change